### PR TITLE
fix: Don't list the target branch

### PR DESCRIPTION
**Pull request by estib-vega**
Don't list the target branch in the sidebar

### CODE REVIEW

**Review by ndom91 — APPROVED** [review]

File: crates/gitbutler-branch-actions/src/branch.rs [hunk 1]
```diff
@@ -343,6 +349,17 @@ impl GroupBranch<'_> {
         }
         .map(BranchIdentity::from)
     }
+
+    /// Determines if the branch is a remote branch by ref name
+    fn is_remote_branch(&self, ref_name: &RemoteRefname) -> bool {
+        if let GroupBranch::Remote(branch) = self {
+            let remote_branch = ref_name.to_string();
```
Comment (Byron) [inline crates/gitbutler-branch-actions/src/branch.rs]: There should never be a need to convert `to_string()`. Maybe there is an `as_ref().as_bstr()` for that?

**Review by Byron — COMMENTED** [review]
Maybe this is related to [this comment](https://github.com/gitbutlerapp/gitbutler/issues/5229#issuecomment-2434644064)?

File: crates/gitbutler-branch-actions/src/branch.rs (hunk 1 above)
Comment (estib-vega) [reply crates/gitbutler-branch-actions/src/branch.rs]: Will update that 👍

Comment (estib-vega) [reply crates/gitbutler-branch-actions/src/branch.rs]: There wasn't but maybe `format` is a bit better?
[see](https://github.com/gitbutlerapp/gitbutler/pull/5289/files#diff-beb9965bba602a6a2e20a58f347bc759f37bf01cd2f9826374f34636c4bfb44dR103-R106)
```rs

impl PartialEq<BStr> for Refname {
    fn eq(&self, other: &BStr) -> bool {
        format!("refs/remotes/{}/{}", self.remote, self.branch) == *other
    }
}
```

Comment (Byron) [reply crates/gitbutler-branch-actions/src/branch.rs]: It still allocates, so I posted #5303 .

Comment (estib-vega) [reply crates/gitbutler-branch-actions/src/branch.rs]: Thanks!

**Comment by Byron** [comment]
Oh, right, I thought/hoped that would fix the issue it was posted in. But even if not, I think thanks to your help it's a bit closer.

**Pull request merged**